### ISSUE TITLE
feature issues

### ISSUE THREAD

**Issue by GoodWing** [kaisunresearch/dash#1]
Can I take a look at the code for creating feature.json，I have already installed morfessor, but I don't know how to use it

**Comment by accreator** [comment]
> Can I take a look at the code for creating feature.json

The complexity of the dependencies makes it tricky to provide easy-to-use scripts in a sharable state that can do feature extraction from scratch (e.g., downloading DNS Census 2013 requires a BitTorrent client). As such,  we choose to provide pre-extracted features directly for now, which should be sufficient for reproducing the results in the paper.

> I have already installed morfessor, but I don't know how to use it

Regarding morfessor, you may follow this example: https://stackoverflow.com/questions/54802849/word-tokenization-when-there-is-no-space/54802923#54802923

**Comment by GoodWing** [comment]
Yes, I know feature extraction from scratch can be cumbersome, but I want to use your model to predict several individual domain names. However, when calling your model, I need to perform feature extraction on my domain name, so I would like to request the construction code for feature.json
